rclcpp/include/rclcpp/timer.hpp

```diff
@@ -58,6 +58,11 @@ class TimerBase
   void
   cancel();
 
+  /// Return the timer cancellation state.
```

Comment:
Would be nice to specify in this doc this may throw a std::runtime_error if the timer state cannot be obtained.

Reply:
Ok, can do that. I'll look for some examples in the codebase.